### PR TITLE
first version of jv-dao-service homework

### PR DESCRIPTION
**Pull request by oleksiistepaniak**

### CODE REVIEW

**Comment by V-Vasylchuk** [comment]
Magnificent!

File: src/main/java/mate/jdbc/Main.java [hunk 1]
```diff
@@ -1,7 +1,81 @@
...
+
+        // TESTING CREATE METHOD
+        Driver thirdDriver = new Driver();
+        thirdDriver.setName("Bob");
+        thirdDriver.setLicenseNumber("2902");
+        Driver driverBob = driverService.create(thirdDriver);
+        Manufacturer thirdManufacturer = new Manufacturer();
+        thirdManufacturer.setCountry("Lithuania");
+        thirdManufacturer.setName("third_name");
+        Manufacturer manufacturerFromLithuania = manufacturerService.create(thirdManufacturer);
```
Comment (Andriy-dz) [inline src/main/java/mate/jdbc/Main.java:47]:
```suggestion
        // TESTING DRIVER CREATE METHOD
        Driver firstDriver = new Driver();
        firstDriver.setName("Alex");
        firstDriver.setLicenseNumber("1224");
        Driver driverAlex = driverService.create(firstDriver);
        
        Driver secondDriver = new Driver();
        secondDriver.setName("Alice");
        secondDriver.setLicenseNumber("1240");
        Driver driverAlice = driverService.create(secondDriver);

        Driver thirdDriver = new Driver();
        thirdDriver.setName("Bob");
        thirdDriver.setLicenseNumber("2902");
        Driver driverBob = driverService.create(thirdDriver);
        
        // TESTING MANUFACTURER CREATE METHOD
        Manufacturer firstManufacturer = new Manufacturer();
        firstManufacturer.setCountry("Ukraine");
        firstManufacturer.setName("first_name");
        Manufacturer manufacturerFromUkraine = manufacturerService.create(firstManufacturer);

        Manufacturer secondManufacturer = new Manufacturer();
        secondManufacturer.setCountry("Spain");
        secondManufacturer.setName("second_name");
        Manufacturer manufacturerFromSpain = manufacturerService.create(secondManufacturer);
       
        Manufacturer thirdManufacturer = new Manufacturer();
        thirdManufacturer.setCountry("Lithuania");
        thirdManufacturer.setName("third_name");
        Manufacturer manufacturerFromLithuania = manufacturerService.create(thirdManufacturer);
```
so it reads a little better, agree) I suggest doing the same with the following cases (Optional)

File: src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).orElseThrow();
```
Comment (Andriy-dz) [inline src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java:22]: I suggest doing the same with the following cases

File: src/main/java/mate/jdbc/service/impl/ManufacturerServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return manufacturerDao.get(id).orElseThrow();
```
Comment (Andriy-dz) [inline src/main/java/mate/jdbc/service/impl/ManufacturerServiceImpl.java:22]: same

File: src/main/resources/init_db.sql [hunk 4]
```diff
@@ -7,3 +7,12 @@ CREATE TABLE `manufacturers` (
                                         `country` VARCHAR(225) NOT NULL,
                                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
                                         PRIMARY KEY (`id`));
+
+CREATE TABLE `drivers` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
```
Comment (Andriy-dz) [inline src/main/resources/init_db.sql:12]:
```suggestion
  `id` BIGINT NOT NULL AUTO_INCREMENT,
```
I suggest doing the same with the following cases

**Review by Andriy-dz — CHANGES_REQUESTED** [review]
Good Job! Let`s improve your solution)

**Review by sarakhmen — APPROVED** [review]
Good job ;)

**Pull request closed without merging**